### DISCUSSION TITLE
Space Discretization

### DISCUSSION THREAD

**Discussion by LTMeyer** [PolymathicAI/the_well#12] [Q&A] [answered]
What space discretization did you use to generate the datasets?

**Comment by mikemccabe210** [comment, marked as answer]
This was largely driven by the domain expert and the numerical solver used for the simulations and specifics can be found in the individual dataset readmes. Some used pseudospectral methods (Fourier or Chebychev), some used finite volume over uniform grids, some used Smoothed-particle Hydrodynamics (SPH) so are technically lagrangian methods interpolated onto a grid. 

Ultimately, all were interpolated onto a uniform grid in some coordinate space where most are in Euclidean space though a couple are spherical or curvilinear. For pseudospectral methods, this was done by simply evaluating the basis functions at the uniform grid points. For FV, the simulations themselves are on uniform grids so the simulation grid is used directly.
